examples/timeseries/qscan.py

```diff
@@ -42,12 +42,17 @@
 data = TimeSeries.fetch_open_data('H1', 1126259446, 1126259478)
 
 # Next, we generate the `~TimeSeries.q_transform` of these data:
-qspecgram = data.q_transform()
+outseg = (1126259462.2, 1126259462.5)
+qspecgram = data.q_transform(outseg=outseg)
```

Comment:
@alurban, can you merge these into a single line?

Reply:
@duncanmmacleod, aye, this is done in the latest commit.